docs/SetupGuide_DotnetOutOfProc.md

```diff
@@ -35,6 +36,28 @@ Please refer to the functions documentation [here](https://learn.microsoft.com/a
 - There's also no direct support for types inherited from underlying service SDKs, such as SqlCommand. Instead, bindings rely on strings, arrays, and serializable types, such as plain old class objects (POCOs).
 - For HTTP triggers, you must use HttpRequestData and HttpResponseData to access the request and response data. This is because you don't have access to the original HTTP request and response objects when running out-of-process.
 
+## Setup Function App
```

Comment:
This isn't a function app though - that's what you create in Azure that you deploy the functions too. Isn't this just instructions for setting up a .NET Isolated function project?

Reply:
It is definitely a part of creating the function. You want me to rather have it as Setup Function project?

Reply:
I would say `Setup Function Project` instead - that's really what you're doing here. (and then update the rest of the text to reflect that as well)

Reply:
And actually this is the case for all the language guides. So we should fix it everywhere. If you could do that all here that'd be great, otherwise you can fix up this one and then we can do a follow-up PR to update the rest.